mix.exs

```diff
@@ -29,6 +29,7 @@ defmodule Faker.Mixfile do
       {:earmark, ">= 0.0.0", only: :dev, runtime: false},
       {:credo, ">= 0.0.0", only: [:dev, :test], runtime: false},
       {:dialyxir, ">= 0.0.0", only: [:dev], runtime: false},
+      {:geohash, "~> 1.0", optional: true},
```

Comment:
@tobyhinloopen what is your opinion?

Reply:
I never actually used optional dependencies, but the docs say this:

```
:optional - marks the dependency as optional. In such cases, the current project will always include the optional dependency but any other project that depends on the current project won’t be forced to use the optional dependency. However, if the other project includes the optional dependency on its own, the requirements and options specified here will also be applied.
```

I'd say optional dependencies are fine. Since the test-suite will always include the dependency, we don't know what happens when the dependency isn't there though. I can imagine if there is a compile-time reference somewhere to the optional dependency, something is going to go down.